test/disable-request-logging.test.js

```diff
@@ -54,17 +54,15 @@ t.test('use a custom instance of \'undici\'', async t => {
       }
     })
 
-    instance.listen({ port: 0 }, (err) => {
+    instance.listen({ port: 0 }, async (err) => {
```

Comment:
still needs updating.

Reply:
yep, that's why I moved it to draft.
I'll ping you once I finish